arcgis-ios-sdk-samples/Content Display Logic/Controllers/SourceCodeViewController.swift

```diff
@@ -38,16 +39,25 @@ class SourceCodeViewController: UIViewController, UIAdaptivePresentationControll
         contentView.addSubview(webView)
         self.webView = webView
         
-        if let filename = filenames.first {
+        if let filename = currentFilename {
+            loadHTMLPage(filename: filename)
+        } else if let filename = filenames.first {
+            currentFilename = filename
             loadHTMLPage(filename: filename)
         }
     }
     
+    /// Change the highlight.js rendered HTML when switch  between modes.
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        loadHTMLPage(filename: currentFilename)
+    }
+    
```

Comment:
This ensures the css get loaded every time we swap the source files, so that the source code pane always be in correct color mode.